SurveyApp/Extensions/UIKit/UITextField/UITextField+Padding.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  UITextField+Padding.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/8/21.
+//
+
+import UIKit
+
+extension UITextField {
+    
+    func setPadding(left: CGFloat, right: CGFloat? = nil) {
```

Comment:
Hmm, I think we will lose the padding if we override `leftView` or `rightView` from outside. 
So, what do you think about going for a different approach? Like this [one](https://www.advancedswift.com/uitextfield-with-padding-swift/) 🤔

Reply:
Thank you for a better approach! I'll make that change 😄